modules/org/infinispan/distribution/native/added/bin/common.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+
+# Use --debug to activate debug mode with an optional argument to specify the port.
+# Usage : server.sh --debug
+#         server.sh --debug 9797
+# By default debug mode is disabled.
+DEBUG_MODE="${DEBUG:-false}"
```

Comment:
I am not sure it matters, but a lot of the settings in here won't work with the native image like debug or jmx and possibly gc log/settings.

Reply:
This is a trimmed down version of the `common.sh` packaged with the server `.zip`. I stripped out a log of the irrelevant parts of the script, but I kept the parsing of `--debug` to stop it being passed as an arg to the server if a user adds it.

I also kept the JMX stuff because I know there's partial support for it in GraalVM now. So figured if we do eventually support it in some form on the native server, there's no need to re-add anything in the image at least. I assumed if the properties are set now they'll just be ignored right?

Reply:
alright. They should be ignored, but tbh I don't know what would happen if passed, I never tested it. GraalVM could puke